azure/Guidelines.md

```diff
@@ -0,0 +1,243 @@
+# Microsoft Azure REST Design Guidelines
+
+## History
+
+| Date | Version | Notes |
+| 2020-Mar-31 | v3.1 | Merge of Azure guidelines with breaking changes update |
```

Comment:
Consider adding something to the effect of "1st public release of guidelines" .

Reply:
Yes.